File: java/01-commands/fix/sourcecode/src/main/java/com/training/app/MyClassExample4.java
package com.training.app;

interface IList {
    int getCount();
    void setCount(int count);
    void counter();
}

interface ICounter {
    double getCount();
    void setCount(double count);
}

interface IListCounter extends IList, ICounter {}

class MyClass {
    void test(IListCounter x) {
        x.setCount(1);
    }

    public static void main(String[] args) {}
}

File: dotnet/01-commands/fix/DemoFix/MyClassExample4.cs
using System;
namespace MyNamespace
{
    interface IList
    {
        int Count
        {
            get;
            set;
        }
        void Counter();
    }
    interface ICounter
    {
        double Count
        {
            get;
            set;
        }
    }

    interface IListCounter : IList, ICounter { }

    class MyClass
    {
        void Test(IListCounter x)
        {
            x.Count = 1;    
        }

        public static void Main() { }
    }
}